src/restapi/fIncomingWebhooks.ts

```diff
@@ -1,9 +1,11 @@
 import {Post} from 'mattermost-redux/types/posts';
 
 import {Request, Response} from 'express';
+import {AppContextWithBot} from 'types/apps';
 import {AppContext} from 'mattermost-redux/types/apps';
```

Comment:
Is this needed any longer?

Reply:
There are multiple new custom context types that are used by the call functions.